Scripts/RemediationScripts/Get-ManagementCertificates.ps1

```diff
@@ -0,0 +1,352 @@
+<###
+# Overview:
+    This script is used to get Management Certificates in a Subscription.
+
+# Prerequisites:
+    1. Co-Admin or higher privileges in a Subscription.
```

Comment:
Just call out Co-Administrator role is required.

Reply:
Added.